### DISCUSSION TITLE
Zen mode.

### DISCUSSION THREAD

**Discussion by dceit** [ppy/osu#26513] [Ideas] [answered]
Similar to Lichess and Chess.com's implementation. A "zen" setting you can enable to hide leader boards, ranked status, PP, or any other competitive / anxiety inducing features. There is a a fair size of the community who prefers to just enjoy the game rather than competing.

There have been skins in stable which use empty elements for combo / score numbers to mimic this, but it's a very hacky workaround. Thoughts on having an official version of this in the future?

**Comment by bdach** [comment, marked as answer]
probably will naturally be possible as we bring the skinning system closer to where we likely want it to eventually be (you'll just be able to delete what annoys you).

**Comment by peppy** [comment]
Shift-tab?
